sesman/session.c

```diff
@@ -89,6 +89,21 @@ dumpItemsToString(struct list *self, char *outstr, int len)
     return outstr ;
 }
 
+/* compare two strings containing ip-addresses and ports but ignore the latter */
+static int
+is_equal_ip_ignoring_port(const char *ip1, const char *ip2, size_t n)
```

Comment:
`const size_t n` ?

Reply:
Ouch, I accidentally used size_t as I always do. But in xrdp's codebase is hardly used. Should I change it to an signed const int here, too?